views/works-page.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/public/styles.css" rel="stylesheet" type="text/css" />
+    <title>works</title>
+  </head>
+  <body>
+    <div class="navbar-nav">
+        <a class="nav-item nav-link active" href="#">Home </a>
+        <a class="nav-item nav-link" href="#">About</a>
+        <a class="nav-item nav-link" href="#">Works</a>
+      </div>
+    </nav>
+    <div class="container2">
+    <div><img class ="album-cover" src="/public/images/trajectory.jpg" alt="trajectory">
+    <p class="text" >Token89 <br> Year: 2019</p>
+    </div>
+    <div><img class ="album-cover" src="/public/images/magma.jpg" alt="magma">
+        <p class="text" >Token79 <br> Year: 2018</p>
+    </div>
+    <div><img class ="album-cover" src="/public/images/cathedral.jpg" alt="cathedral">
+        <p class="text" >Token <br> Year: 2013</p>
+    </div>
+    <div><img class ="album-cover" src="/public/images/vaudeville.jpg" alt="vaudeville">
```

Comment:
here again, you don't need the /public/ here, just the path after it => src="/images/cathedral.jpg"